### ISSUE TITLE
Tidy up testdata

### ISSUE THREAD

**Issue by rsheeter** [googlefonts/fontations#1127]
@drott [suggested](https://github.com/googlefonts/fontations/pull/1099#discussion_r1723408894):

> Similar to practices in Chromium or our other open source projects, I would suggest to create a `test_data/third_party/` or `test_data/third_party/harfbuzz` directory, and please a readme with provenance and license there, so we have a clear indication where the font is coming from. 

The original comment was for one specific test file but I think it applies to all our test data.

**Comment by rsheeter** [comment]
@drott do you think this is required for Chrome launch? - waffled over whether to mark that milestone.

**Comment by drott** [comment]
> do you think this is required for Chrome launch? 

No, but should happen rather sooner than later, as it becomes more difficult to figure out what came from where over time.
